tutor/commands/k8s.py

```diff
@@ -590,12 +544,10 @@ def k8s_namespace(config: Config) -> str:
 k8s.add_command(delete)
 k8s.add_command(init)
 k8s.add_command(scale)
-k8s.add_command(createuser)
-k8s.add_command(importdemocourse)
-k8s.add_command(settheme)
 k8s.add_command(exec_command)
 k8s.add_command(logs)
 k8s.add_command(wait)
 k8s.add_command(upgrade)
 k8s.add_command(apply_command)
 k8s.add_command(status)
+jobs.add_job_commands(do)
```

Comment:
I found that custom commands cannot be added, because this line is run before plugin modules are imported. I was able to fix it by moving this line into a `Actions.PLUGINS_LOADED` callback.

Reply:
Right. I confess that I did not even take the time to try out the API...